EIPS/eip-4469.md

```diff
@@ -0,0 +1,63 @@
+---
+eip: 4469
+title: Wrapped metaverse token
+author: Hyungsuk Kang (@hskang9)
+discussions-to: https://github.com/ethereum/EIPs/issues/4469
```

Comment:
A forum post at https://ethereum-magicians.org/ is very strongly preferred as a discussions-to link.  We are trying to get all EIP discussion mo move over to there rather than GitHub.  You can login with your GitHub account, so no need to go through any account creation process.

Reply:
That is good to hear. It would be better to comment and discuss. Here is the link https://ethereum-magicians.org/t/eip-4469-wrapping-erc1155-into-erc20s/7516